src/Inventory.zig

```diff
@@ -657,6 +673,15 @@ pub const Command = struct { // MARK: Command
 					if(info.previousDurability <= info.durability) {
 						info.item.deinit();
 					}
+				},
+				.addHealth => |info| {
+					if (side == .server) {
+						if (info.source.?.player.health <= 0) {
+							info.cause.sendMessage(info.source.?.name);
+							
+							main.network.Protocols.genericUpdate.sendKill(info.source.?.conn);
```

Comment:
Why send it to the client, if the health is properly synchronized you can just do the same check on the client side.